draft-ietf-privacypass-protocol.md

```diff
@@ -167,10 +167,19 @@ defined in {{tokenkeys-values}}.
 | token-key    | The base64url encoding of the Public Key for use with the issuance protocol, including padding, represented as a JSON string ({{RFC8259, Section 7}}) |
 {: #tokenkeys-values title="Issuer 'token-keys' object description'"}
 
-Issuers MAY advertise multiple token-keys for the same token-type to
-support key rotation. In this case, Issuers indicate preference for which
-token key to use based on the order of keys in the list, with preference
-given to keys earlier in the list.
+Each "token-keys" JSON object may also contain the optional field "not-before".
+The value of this field is the UNIX timestamp (number of milliseconds since
+January 1, 1970, UTC) at which the key can be used. If this field is present,
+Clients SHOULD NOT use a token key before this timestamp, as doing so may
+lead to issuance failures. The purpose of this field is to assist in scheduled
+key rotations.
+
+Beyond staging keys with the "not-before" value, Issuers MAY advertise multiple
+"token-keys" for the same token-type to facilitate key rotation. In this case,
+Issuers indicate preference for which token key to use based on the order of
+keys in the list, with preference given to keys earlier in the list. Clients
+SHOULD use the first key in the "token-keys" list that either does not have a
+"not-before" value or has a "not-before" value in the past.
```

Comment:
Do you want to explicitly call out origin behavior, where they should attempt to verify using all present keys, or add some skew around the `not-before` for verification?

I'm worried an origin will use the not-before as a hard cutoff, and clients with positive skew will use this before origins are allowing verification on it.

Reply:
@smhendrickson done! How does it look?

Reply:
LGTM